docs/_plugins/copy_api_dirs.rb

```diff
@@ -150,4 +150,31 @@
     cp("../R/pkg/DESCRIPTION", "api")
   end
 
+  if not (ENV['SKIP_SQLDOC'] == '1')
+    # Build SQL API docs
+
+    puts "Moving to project root and building API docs."
+    curr_dir = pwd
+    cd("..")
```

Comment:
Rather than `cd`, is it possible to generate the output directly into `docs/` somewhere? maybe I miss why that's hard. It would avoid creating more temp output dirs in the `sql` folder